test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -2028,5 +2029,18 @@ public void CommandTrainingBinaryFieldAwareFactorizationMachineWithValidationAnd
             Assert.True(outputPath.CheckEqualityNormalized());
             Done();
         }
+
+        [Fact]
+        public void DataTypes()
+        {
+            //Skip for linux because DATE/TIME format is different.
```

Comment:
Can we instead write the DATE/TIME using a stable format that works across all platforms?